test/service_manager/service_manager.go

```diff
@@ -0,0 +1,156 @@
+package service_manager
+
+import (
+	"github.com/Shopify/sarama"
+	"github.com/strimzi/strimzi-canary/internal/config"
+	"log"
+	"os"
+	"os/exec"
+	"time"
+)
+
+// Implementation of Service Manager
+type ServiceManager struct {
+	CanaryConfig
+	Paths
+}
+
+// Configurations of canary that is manipulated in e2e tests
+type CanaryConfig struct {
+	RetentionTime      string
```

Comment:
Retention? Isn't it the Reconcile time? As I see it's set on canary? This should be changed in the service manager as well.